PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/Players/OoyalaPlayerTokenPlayerViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  OoyalaPlayerTokenPlayerViewController.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import Foundation
+
+class OoyalaPlayerTokenPlayerViewController: OOOoyalaTVPlayerViewController, OOEmbedTokenGenerator {
+  
+  var option: PlayerSelectionOption!
+  var embedCode: String!
+  var pcode: String!
+  var playerDomain: String!
+  var apiKey: String!
+  var secret: String!
+  var authorizeHost: String!
+  var accountId: String!
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    self.pcode = self.option.pcode
+    self.playerDomain = self.option.domain
+    self.authorizeHost = "http://player.ooyala.com"
+    
+    /*
+     * The API Key and Secret should not be saved inside your applciation (even in git!).
+     * However, for debugging you can use them to locally generate Ooyala Player Tokens.
+     */
+    self.apiKey = "Fill me in"
+    self.secret = "Fill me in"
+    self.accountId = "Fill me in"
+    
+    self.player = OOOoyalaPlayer(pcode: self.pcode, domain: OOPlayerDomain(string: self.playerDomain), embedTokenGenerator: self)
+    self.player.setEmbedCode(self.option.embedCode)
+    self.player.play()
+  }
+  
+  func token(forEmbedCodes embedCodes: [Any]!, callback: OOEmbedTokenCallback!) {
+    var params = NSMutableDictionary()
+    
+    params["account_id"] = self.accountId
+    
+    let uri = String(format: "/sas/embed_token/%@/%@", self.pcode, (embedCodes.map{$0 as! String}).joined(separator: ","))
```

Comment:
Try to cast _embedCodes_ from `[Any]!` to `[String]`

Reply:
Done